ops/review-deploy.tmpl.yaml

```diff
@@ -40,12 +40,46 @@ ingress:
         - path: /
 
 extraEnvVars: &envVars
+  - name: CONFDIR
```

Comment:
delete this file, its for hyku to deploy to r2-staging.

Reply:
keeping so we are inline with hyku